body.tex

```diff
@@ -1,166 +1,464 @@
 
-\section{Some technicalities: author list and standard LSST references} 
+With respect to the database, we discussed the possibility of having local registries within individual jobs.
+Each job has a fast local SQLite registry.
+After each job finishes, the indiviual registry is then collated into the central database.
+Besides simplifying failure recovery, another benefit is to avoid potential bottlenecks related to large scale parallel SQL transactions by bundling multiple transactions together and handling database connections in a centralized manner.
+This envisions a fully shared-nothing architecture, but implementing this requires more thorough design work in conjunction with the Generation 3 Middleware development.
+Also, the current implementations of the Butler registry depend on conflict resolution syntax to resolve insertions of complexs datasets, which may not work in this scenario.
 
-Thank you Tim Jenness and Wil O'Mullane for helping with templates! 
 
-\subsection{The LSST LaTeX Classes}
+\section{Approaches and strategies}
 
-Please see the installation instructions\footnote{\url{https://lsst-texmf.lsst.io/install.html}} 
-for lsst-texmf. Once you have it installed, you should be able to compile your paper
-using make. 
+\poc's goal is to demonstrate that running Data Release Production (DRP) on AWS is possible.
+The following approaches were taken.
 
-\subsection{How to handle author list?} 
+\subsection{Progress in phases}
 
-Authors come from the authors.yaml file --  find the author ids in the lsst-texmf/etc/authordb.yaml - use db2authors to get the authors and institutes from the db. 
+\citeds{DMTN-114} proposed development phases that broadly outlined how progress can be made and when performance would be monitored. The outlined plan was to move the execution to AWS without any backend modifications first and then gradually switch out each component to AWS.
 
-{\bf XXX Wil, the above is unclear: need more detail about how to use db2authors,
-what is its output and what to do with it...} 
+Before this \poc, the DRP execution relied a shared POSIX-compliant filesystem.
+In the first phase such filesystem was replaced by Amazon EFS.
+The data repository, including a POSIX datastore and a SQLite registry, was entirely located on EFS that was mouted to each of the EC2 compute instances.
 
+Later, the S3 datastore replaced the need of EFS, and the PostgreSQL RDS replaced the SQLite registry.
```

Comment:
"Subsequently, support for the S3 datastore eliminated the need for EFS, ..."

Reply:
Was fixed but left open, resolved.